flopy/utils/gridintersect.py

```diff
@@ -468,6 +475,8 @@ def _intersect_point_shapely(self, shp, sort_by_cellid=True):
         sort_by_cellid : bool, optional
             flag whether to sort cells by id, used to ensure node
             with lowest id is returned, by default True
+        keep_stacked_cells : bool, optional
+            For shapely only, keeps all intersected cells even if cell vertices are already in the results.
```

Comment:
`keep_stacked_cells` doesn't quite capture the full behavior of this option. Even without stacked cells, setting this option to True will e.g. return 2 intersection results for a point or linestring on a boundary between two cells. 

also ensure <80 characters per line in docsstring

Reply:
I like `keep_all_intersections` and `return_all_intersections`. If I had to pick I would chose `return_all_intersections`.